nengo/spa/vocab.py

```diff
@@ -81,6 +81,8 @@ def __init__(self, dimensions, randomize=True, unitary=False,
         self.include_pairs = include_pairs
         self._identity = None
         self.rng = rng
+        self.read_only = False
+        self.parent = None
```

Comment:
@tbekolay There. I removed the cyclic reference. Check the ```transform_to``` function to see if that will freak out the garbage collector.